### PR TITLE
PSQLADM-355 Add a contributing guide for documentation

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/conf.py
added: source/contributing.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+For more information about ProxySQL and the ProxySQL Admin tool, see the [ProxySQL and ProxySQL Admin documentation](https://www.percona.com/doc/proxysql/index.html)
+
+## Installation
+
+You can install ProxySQL and the ProxySQL Admin tool using the following methods:
+- Use the Percona repository
+- Building from the [source code](https://github.com/percona/proxysql-admin-tool)
+
+The ProxySQL and ProxySQL Admin tool documentation source is [here](https://github.com/percona/proxysql-admin-tool-doc)
+## ProxySQL Information
```
Comment (nastena1606) [inline README.md:14]: An extra blank line should go here

File: source/contributing.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+- Click the **Submit DOC bug** link on the sidebar. This opens the [Jira issue tracker](https://jira.percona.com/projects/PSQLADM/issues) for the doc project.
+- Sign in (create a Jira account if you don't have one) and click **Create** to create an issue.
+- In the following fields, describe the issue:
+
+  - In the Summary, provide a brief description of the issue
+  - In the Description, provide more information about the issue, along with a Steps To Reproduce section, if needed. In the Affects Version/s field, enter the version number. It is OK to add a version number with an ".x" (such as `2.0.x`).
+
+## Contribute to documentation yourself
+
+In the sidebar, there is a **This Page** section. The **Edit in GitHub** link opens the source file of the page on GitHub. You make changes, and merge your pull request into the documentation to documentation, learn about the following:
```
Comment (nastena1606) [inline source/contributing.md:23]:
```suggestion
In the sidebar, there is a **This Page** section. The **Edit in GitHub** link opens the source file of the page on GitHub. You make changes, and merge your pull request into the documentation. As a precondition to contributing  to documentation, learn about the following:
```

File: source/contributing.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+   git checkout main && git pull origin main
+  ```
+
+5. Create a separate branch for your changes
+
+  ```sh
+   git checkout -b <my_changes>
+  ```
+
+6. Do your work. Add code examples, if necessary. We recommend that you build an HTML file, either [locally](#Install Sphinx and build locally) or [using Docker](# Use Docker) to view your changes. You can run the command to `make clean html` to see the latest changes. If you have questions on formatting, use a [cheat sheet](https://sphinx-tutorial.readthedocs.io/cheatsheet/?highlight=-b#rst-cheat-sheet).
```
Comment (nastena1606) [inline source/contributing.md:89]: Links won't work in this folrmat. They require no spaces between # and the link text, plust use - as a separator

File: source/contributing.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+1. Install [pip](https://pip.pypa.io/en/stable/installing/)
+2. Install [Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html).
+3. While in the root directory of the doc project, run the following command to build the documentation:
+
+  ```sh
+   make clean html
+  ```
+
+This command does two tasks. The `clean` command rebuilds the `/build` directory and the `latexpdf` commands creates a PDF document. The PDF document is stored in the `/build/html` folder.
```
Comment (nastena1606) [inline source/contributing.md:157]: This sentence is out of place. It tells about PDFs in the middle of the build html section.

File: README.md (hunk 1 above)
Comment (patrickbirch) [reply README.md:14]: Fixed.

File: source/contributing.md (hunk 2 above)
Comment (patrickbirch) [reply source/contributing.md:23]: Updated text.

File: source/contributing.md (hunk 4 above)
Comment (patrickbirch) [reply source/contributing.md:157]: Fixed.

File: source/contributing.md (hunk 3 above)
Comment (patrickbirch) [reply source/contributing.md:89]: Fixed.

File: README.md [hunk 5]
```diff
@@ -1 +1,43 @@
-# proxysql-admin-tool-doc
\ No newline at end of file
+# ProxySQL Admin Documentation
+
+![Percona logo](source/percona-logo.jpg)
+
+For more information about ProxySQL and the ProxySQL Admin tool, see the [ProxySQL and ProxySQL Admin documentation](https://www.percona.com/doc/proxysql/index.html)
```
Comment (alina-derkach-oaza) [inline README.md:5]: add a full stop

File: README.md [hunk 6]
```diff
@@ -1 +1,43 @@
...
+
+You can install ProxySQL and the ProxySQL Admin tool using the following methods:
+- Use the Percona repository
+- Building from the [source code](https://github.com/percona/proxysql-admin-tool)
+
+The ProxySQL and ProxySQL Admin tool documentation source is [here](https://github.com/percona/proxysql-admin-tool-doc)
+
+## ProxySQL Information
+
+You can also install ProxySQL from source code [here](https://github.com/sysown/proxysql)
```
Comment (alina-derkach-oaza) [inline README.md:17]: from THE source code

File: README.md [hunk 7]
```diff
@@ -1 +1,43 @@
...
+# ProxySQL Admin Documentation
+
+![Percona logo](source/percona-logo.jpg)
+
+For more information about ProxySQL and the ProxySQL Admin tool, see the [ProxySQL and ProxySQL Admin documentation](https://www.percona.com/doc/proxysql/index.html)
+
+## Installation
+
+You can install ProxySQL and the ProxySQL Admin tool using the following methods:
+- Use the Percona repository
```
Comment (alina-derkach-oaza) [inline README.md:10]: Using?

File: README.md [hunk 8]
```diff
@@ -1 +1,43 @@
...
+- Building from the [source code](https://github.com/percona/proxysql-admin-tool)
+
+The ProxySQL and ProxySQL Admin tool documentation source is [here](https://github.com/percona/proxysql-admin-tool-doc)
```
Comment (alina-derkach-oaza) [inline README.md:13]: add a full stop

File: README.md (hunk 5 above)
Comment (patrickbirch) [reply README.md:5]: Done

File: README.md (hunk 7 above)
Comment (patrickbirch) [reply README.md:10]: rewritten

File: README.md (hunk 6 above)
Comment (patrickbirch) [reply README.md:17]: added

File: source/contributing.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+- Select the **Submit DOC bug** link on the sidebar. This action opens the [Jira issue tracker](https://jira.percona.com/projects/PSQLADM/issues) for the doc project.
+
+- Log in (create a Jira account if you don't have one) and select **Create** to open the Jira form.
+  
+![Complete the Jira form](_images/add-jira.png)
+
+- In the following fields, describe the issue:
+
+  - In the Summary, provide a brief description of the issue
```
Comment (alina-derkach-oaza) [inline source/contributing.md:29]: add a full stop

**Review by alina-derkach-oaza — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (patrickbirch) [reply README.md:13]: Done.

File: source/contributing.md (hunk 9 above)
Comment (patrickbirch) [reply source/contributing.md:29]: Done.

**Pull request merged**